SUMMARY.md

```diff
@@ -1,22 +1,13 @@
 # Table of contents
 
 * [Angle Documentation Portal](README.md)
+* [⚒ Developers](https://developers.angle.money)
+
+## Protocol Overview
+
 * [📖 Whitepapers](whitepaper.md)
 * [📚 Glossary](glossary.md)
 * [❓ FAQ](faq.md)
```

Comment:
We should update this FAQ because it just has questions related to the old module

Reply:
I was thinking about having one for each module

Reply:
Yes but here it's in the Protocol overview (and not in the COre Module overview)